tutorials/lodash/difference.md

```diff
@@ -0,0 +1,43 @@
+The `difference()` function in lodash takes two arguments, the array in question and an array of values to compare against, and returns a new array.
+
+```javascript
+const _ = require('lodash');
+const array = [1, 2, 3, 4, 5];
+const values = [1, 2, 3];
+_.difference(array, values); // returns [4, 5]
+```
+
+## differenceBy()
+
+The `differenceBy()` function behaves similarly to the `difference()` function with the addition of a third argument that transforms each element before comparison.
```

Comment:
The `differenceBy()` function accepts a third argument, a callback function that transforms each element before calculating the difference. For example, below is how you can calculate the difference between two sets of numbers by their `Math.floor()` value.